test/integration/traces_test.go

```diff
@@ -752,7 +752,7 @@ func testNestedHTTPTracesKProbes(t *testing.T) {
 
 		// Check the information of the java parent span
 		res = trace.FindByOperationName("GET /jtrace")
-		require.Len(t, res, 1)
+		require.Len(t, res, 2)
```

Comment:
Ah right, I guess now multiple spans will match with the same URL path. I think we need to extend FindOperationByName to use an optional parameter which is the span kind and look for the server span. Otherwise many other tests will have an issue.